Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/AppCenterContentPage.xaml.cs

```diff
@@ -25,5 +26,12 @@ async void UpdateEnabled(object sender, ToggledEventArgs e)
         {
             await AppCenter.SetEnabledAsync(e.Value);
         }
+
+        void UserIdCompleted(object sender, EventArgs e)
+        {
+            //TODO: Enable this once NuGet package is updated as part of release process.
+            //var text = string.IsNullOrEmpty(UserIdEntryCell.Text) ? null : UserIdEntryCell.Text;
+            //AppCenter.SetUserId(text);
```

Comment:
Note for next steps (not this PR), when you will do the demo update with the new nugets, please also try to uncomment those lines in that next step.

Reply:
ok